src/motion/motion.ts

```diff
@@ -75,7 +75,7 @@ export class Motion implements vscode.Disposable {
         switch (this._motionMode) {
             case MotionMode.Caret:
                 // Valid Positions for Caret: [0, eol)
-                this.position.positionOptions = PositionOptions.CharacterWiseExclusive;
+                this.position.positionOptions = PositionOptions.CharacterWiseInclusive;
```

Comment:
With VsCode, they give you indices on where you are in the sentence. For instance, the sentence, `abc` has valid positions of 0, 1, 2, 3. The intention of the character wise inclusive vs exclusive is to denote whether `3` is an appropriate index. In the case of caret, it is not as you should not be able to have a caret starting from index 3 hence it is an exclusive motion.

Maybe the better way of doing it would be below and maybe have it forgo that inclusive/exclusive check in the `getRight` call?

```
    /**
     * Allows us to simulate a block cursor by highlighting a 1 character
     * space at the provided position in a lighter color.
     */
    private highlightBlock(start: Position): void {
        this.highlightRange(start, start.getRight());
    }
```